packages/relayer/indexer/handle_event.go

```diff
@@ -24,7 +24,7 @@ func (svc *Service) handleEvent(
 
 	// handle chain re-org by checking Removed property, no need to
 	// return error, just continue and do not process.
-	if raw.Removed {
+	if raw.Removed || event.Signal == relayer.ZeroHash {
```

Comment:
Since `message == zeroHash` should not happen in theory, do we need to `log.Warn` something here?

Reply:
It appears to be happening, I noticed today there are some events with signal zero hash being picked up. They only started appearing since our last reboot of the network.

Reply:
After looking in to it, it seems every block has multiple 0 hash signals. Added log.warn for now.